android/.idea/modules.xml

```diff
@@ -5,6 +5,7 @@
       <module fileurl="file://$PROJECT_DIR$/android.iml" filepath="$PROJECT_DIR$/android.iml" />
       <module fileurl="file://$PROJECT_DIR$/../node_modules/astro-sdk/android/astro.iml" filepath="$PROJECT_DIR$/../node_modules/astro-sdk/android/astro.iml" />
       <module fileurl="file://$PROJECT_DIR$/../node_modules/astro-sdk/node_modules/mobify-push-android-client/pushclient/pushclient.iml" filepath="$PROJECT_DIR$/../node_modules/astro-sdk/node_modules/mobify-push-android-client/pushclient/pushclient.iml" />
+      <module fileurl="file://$PROJECT_DIR$/../node_modules/astro-sdk/node_modules/mobify-push-android-client/pushclient/pushclient.iml" filepath="$PROJECT_DIR$/../node_modules/astro-sdk/node_modules/mobify-push-android-client/pushclient/pushclient.iml" />
```

Comment:
Android studio always seems to add this twice. You got me to remove the second occurrence from the sandbox because it was causing someone issues. Should we do that here?